### PR TITLE
Kuaikanmanhua - new extension

### PR DESCRIPTION
**Pull request by SnakeDoc83**
New extension for a Chinese source.

### CODE REVIEW

File: src/zh/kuaikanmanhua/src/eu/kanade/tachiyomi/extension/zh/kuaikanmanhua/Kuaikanmanhua.kt [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+import com.google.gson.JsonArray
+import com.google.gson.JsonObject
+
+class Kuaikanmanhua : ParsedHttpSource() {
+
+    override val name = "Kuaikanmanhua"
+
+    override val baseUrl = "https://www.kuaikanmanhua.com"
+
+    override val lang = "en"
```
Comment (arkon) [inline src/zh/kuaikanmanhua/src/eu/kanade/tachiyomi/extension/zh/kuaikanmanhua/Kuaikanmanhua.kt]: zh

File: src/zh/kuaikanmanhua/src/eu/kanade/tachiyomi/extension/zh/kuaikanmanhua/Kuaikanmanhua.kt [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+        return pages
+    }
+
+    override fun imageUrlParse(document: Document): String = throw UnsupportedOperationException("Not used")
+
+    // Filters
+
+    override fun getFilterList() = FilterList(
+        Filter.Header("NOTE: Ignored if using text search!"),
```
Comment (arkon) [inline src/zh/kuaikanmanhua/src/eu/kanade/tachiyomi/extension/zh/kuaikanmanhua/Kuaikanmanhua.kt]: Might want this to be in Chinese?

File: src/zh/kuaikanmanhua/src/eu/kanade/tachiyomi/extension/zh/kuaikanmanhua/Kuaikanmanhua.kt (hunk 1 above)
Comment (SnakeDoc83) [reply src/zh/kuaikanmanhua/src/eu/kanade/tachiyomi/extension/zh/kuaikanmanhua/Kuaikanmanhua.kt]: Right

File: src/zh/kuaikanmanhua/src/eu/kanade/tachiyomi/extension/zh/kuaikanmanhua/Kuaikanmanhua.kt (hunk 2 above)
Comment (SnakeDoc83) [reply src/zh/kuaikanmanhua/src/eu/kanade/tachiyomi/extension/zh/kuaikanmanhua/Kuaikanmanhua.kt]: Makes sense.

**Pull request merged**